.github/workflows/lint.yml

```diff
@@ -47,3 +47,9 @@ jobs:
         python tools/lint.py --diff-against origin/$GITHUB_BASE_REF
         python tools/unicode-check.py
         python tools/check_test_name.py
+
+    - name: Check that Python.h is first in any file including it.
+      shell: bash
+      run: |
+        python tools/check_python_h_first.py $(find scipy -name highspy -prune \
```

Comment:
If the runtime is not too bad it could also be nice to add this to the `dev.py lint` task so the local tooling reflects what happens in CI

Reply:
Sorry I should have been clearer, I meant here
https://github.com/scipy/scipy/blob/0b949514128c3790666862229d0f956009d63ed1/dev.py#L953-L957
not in the lint.py file itself

Reply:
Okay, done now.